app/scripts/proactive/config.js

```diff
@@ -49,18 +49,19 @@ define(function () {
             'cpython': 'python',
             'ruby': 'ruby',
             'perl': 'perl',
+            'php': 'php',
             'powershell': 'powershell',
             'r': 'text/x-rsrc',
             'vbscript': 'text/vbscript'
         },
         languages_available: {
             'Script/selection': [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
             'Script/environment': [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby"],
-            'Script/pre': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R", "vbscript"],
-            'Script/task': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R", "vbscript"],
-            'Script/post': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R", "vbscript"],
+            'Script/pre': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "php", "powershell", "R", "vbscript"],
+            'Script/task': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "php", "powershell", "R", "vbscript"],
+            'Script/post': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "dockerfile", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "php", "powershell", "R", "vbscript"],
             'Script/flow': [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
-            'Script/clean': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R", "vbscript"]
+            'Script/clean': [" ", "bash", "shell", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "php", "R", "vbscript"]
```

Comment:
"powershell", "php" => "php", "powershell"

Reply:
actually, the list is not alphabetically sorted. Don't know how it should be